src/components/BodyPortal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import { BodyPortalSlotsContext } from './BodyPortalSlotsContext';
+
+const getInsertBeforeTarget = (bodyPortalSlots: string[], slot: string) => {
+  // Note: If the slot is not found in bodyPortalSlots, this code will insert the tag before the first slot
```

Comment:
there are situations, such as rendering an absolutely positioned dialog, when we might not care to specify the slot order and its ok for it to get added wherever. i think typically it would be added at the bottom of the body, but i'm not sure it'll really matter if it goes at the top instead

Reply:
Changed getInsertBeforeTarget to append by default if the slot is not found